packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -238,9 +239,7 @@ export class FilamentViewer extends LitElement {
   private[$render]() {
     this[$rendering] = true;
 
-    if (this[$renderer] != null) {
-      this[$renderer].render(this[$swapChain], this[$view]);
-    }
+    this[$renderer]?.render(this[$swapChain], this[$view]);
```

Comment:
We haven't set up our linter for this yet, but please prefer an explicit `null` check over optional chaining (currently `clang-format` does bad things to optional chaining expressions; I'm actually surprised that our pre-commit hook didn't break this for you).

Reply:
Done!